cfme/tests/automate/test_domain_priority.py

```diff
@@ -38,6 +38,14 @@ def copy_domain(request):
     return domain
 
 
+@pytest.fixture(scope="module")
+def original_domain(request):
+    domain = Domain(name=fauxfactory.gen_alphanumeric(), enabled=True)
+    domain.create()
+    request.addfinalizer(lambda: domain.delete() if domain.exists() else None)
```

Comment:
@mfalesni, this has proven to be both effective and (I think) beautiful for our purposes. I think it would make for a great entry in the [tips and tricks](https://cfme-tests.readthedocs.org/guides/tipsntricks.html) doc, which could stand some fresh attention anyway. :)